cores/esp8266/LwipIntfDev.h

```diff
@@ -0,0 +1,336 @@
+
+#ifndef _LWIPINTFDEV_H
+#define _LWIPINTFDEV_H
+
+// TODO:
+// remove all Serial.print
+// unchain pbufs
+
+#include <netif/ethernet.h>
+#include <lwip/init.h>
+#include <lwip/netif.h>
+#include <lwip/etharp.h>
+#include <lwip/dhcp.h>
+#include <lwip/apps/sntp.h>
+
+#include <user_interface.h>	// wifi_get_macaddr()
+
+#include "SPI.h"
+#include "Schedule.h"
+#include "lwIPIntf.h"
```

Comment:
framework-arduinoespressif8266/cores/esp8266/LwipIntfDev.h:20:10: fatal error: LwIPIntf.h: No such file or directory

```suggestion
#include "LwipIntf.h"
```

Reply:
Thanks, I will update this PR to latest master.

Reply:
Fix pushed, thanks @jbaudoux